CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 #### Upcoming Changes
 
+* fix(BREAKING): Only initialize program builtins instead of a preset list
```

Comment:
As this is a breaking change I think we should mention which part of the public api was changed. I would add something like: Replace `CairoRunner` method `initialize_all_builtins` with `initialize_program_builtins`.

Reply:
Done!